src/openforms/js/components/admin/forms/Inputs.js

```diff
@@ -1,4 +1,5 @@
 import classNames from 'classnames';
+import flatpickr from 'flatpickr';
```

Comment:
Why did this work? Or did we have "undefined is not a function" runtime errors?

Reply:
I think there's an import in the entrypoint that gets installed to `window.flatpickr` which is not executed in storybook and that caused it t work.

Globals... :cry: